where in the world/where in the world/ViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ViewController.swift
+//  where in the world
+//
+//  Created by Li Zhang on 2020-02-11.
+//  Copyright © 2020 Li Zhang. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class MapViewController: UIViewController {
```

Comment:
You declare the `MapViewController` twice using `class MapViewController: ...`. This is not good practice. Only do this once. This file can be deleted, since you've already created a `MapViewController.swift` file, and this functionality should be moved there.